docs/source/usage/plugins/radiation.rst

```diff
@@ -461,7 +459,7 @@ Please be aware that all datasets in the openPMD output are given in the PIConGP
 .. code:: python
 
    import openpmd_api as io
-   series = io.Series("e_radAmplitudes_%T_0_0_0.h5", io.Access_Type.read_only)
+   series = io.Series("e_radAmplitudes_%T.pb", io.Access_Type.read_only)
```

Comment:
The file ending of adios files is `*.bp`

Reply:
good catch - will fix